drivers/sensor/ti/ina226/ina226.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_INA226_INA226_H_
+#define ZEPHYR_DRIVERS_SENSOR_INA226_INA226_H_
+
+#include <stdint.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/sensor.h>
+
+/* Device register addresses. */
```

Comment:
There's no need to have a header as this is all local to the ina226.c compilation unit

Reply:
Ok @teburd. I could change it. But other INAXXX drivers (such as INA219) include a separate header. Kindly suggest how we should proceed.